android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -1589,4 +1697,33 @@ public void onMediaFilesUpdated(Url pageUrl, PlaylistItem[] playlistItems) {
             }
         }
     }
+
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        try {
+            FullscreenManager fullscreenManager =
+                    BraveActivity.getBraveActivity().getFullscreenManager();
+            // Remove the observer when the layout is detached from the window.
+            fullscreenManager.removeObserver(this);
+        } catch (BraveActivity.BraveActivityNotFoundException e) {
+            Log.e(TAG, "BraveActivity not found when entering picture in picture mode.", e);
+        }
+    }
+
+    // FullscreenManager.Observer method.
+    @Override
+    public void onEnterFullscreen(Tab tab, FullscreenOptions options) {
+        if (mYouTubePipTabId == tab.getId()) {
+            try {
+                BraveActivity.getBraveActivity()
```

Comment:
that place as well

Reply:
https://github.com/brave/brave-core/pull/29431#discussion_r2135552600